lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/JavaClrBridge.vcxproj

```diff
@@ -29,6 +29,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/JavaClrBridge.vcxproj should be renamed as 
lang/cpp/reef-bridge-clr/src/main/Cpp/CppBridge/JavaClrBridge/Org.Apache.REEF.Bridge.JavaClrBridge.vcxproj